src/components/experiment-wrapper/index.js

```diff
@@ -11,6 +11,7 @@ import './experiment-wrapper.css';
 const MAX_NUMBER_COMPARISONS = 2; // 0-based, so three
 
 const ExperimentWrapper = ({ theme }) => {
+  const [sidebarVisible, setSidebarVisible] = useState(true);
```

Comment:
A nice convention for booleans is using **is** at the front of variable names. What do you think about the following?

```suggestion
  const [isSidebarVisible, setIsSidebarVisible] = useState(true);
```

Reply:
sure, thanks for the suggestion! I'll make the change